contracts/access/access_control/IAccessControl.sol

```diff
@@ -45,4 +45,27 @@ interface IAccessControl is IAccessControlInternal {
      * @param role role to relinquish
      */
     function renounceRole(bytes32 role) external;
+
+    /**
+     * @dev Returns one of the accounts that have `role`. `index` must be a
+     * value between 0 and {getRoleMemberCount}, non-inclusive.
+     *
+     * Role bearers are not sorted in any particular way, and their ordering may
+     * change at any point.
+     *
+     * WARNING: When using {getRoleMember} and {getRoleMemberCount}, make sure
+     * you perform all queries on the same block. See the following
+     * https://forum.openzeppelin.com/t/iterating-over-elements-on-enumerableset-in-openzeppelin-contracts/2296[forum post]
+     * for more information.
+     */
+    function getRoleMember(
+        bytes32 role,
+        uint256 index
+    ) external view returns (address);
+
+    /**
+     * @dev Returns the number of accounts that have `role`. Can be used
```

Comment:
Use `notice` tags for high-level descriptions and `dev` tags for obscure developer notes.  I don't know if this is the "correct" usage of these tags, but it's how it's done in the rest of SS.